container_linux.go

```diff
@@ -125,7 +126,17 @@ func (c *linuxContainer) commandTemplate(p *Process, childPipe *os.File) (*exec.
 		Path: c.initPath,
 		Args: c.initArgs,
 	}
-	cmd.Stdin = p.Stdin
+	if p.Stdin != nil {
+		stdin, err := cmd.StdinPipe()
+		if err != nil {
+			return nil, err
+		}
+		go func() {
+			io.Copy(stdin, p.Stdin)
+			stdin.Close()
+		}()
+	}
+
```

Comment:
you can take my branch from https://github.com/LK4D4/docker/tree/new_libcontainer_api and integration tests will fail without that fix. I'll try to find author of original code in docker(`execdriver.StdConsole`), maybe he knows what's difference.